frontend/app/module/data_entry/PollingStation/PollingStationLayout.tsx

```diff
@@ -11,15 +11,11 @@ import { Badge, PageTitle, PollingStationNumber, WorkStationNumber } from "@kies
 import { usePollingStationStatus } from "@kiesraad/util";
 
 export function PollingStationLayout() {
-  const { election } = useElection();
+  const { election, pollingStations } = useElection();
```

Comment:
just a thought, but we could add pollingStationId as a param to useElection and have it return the memoized pollingStation instead of having a separate hook.

Reply:
Good idea, changed in c2c9b23003554369a9c4355ad3525255c8d5196d